### PR TITLE
fix windows bug for common lib

### PR DESCRIPTION
**Pull request by risemeup1**
### PR types

Others
### PR changes

Others
### Description

修复QA反馈发版windows编译bug找不到common.lib

### CODE REVIEW

File: cmake/inference_lib.cmake [hunk 1]
```diff
@@ -236,7 +236,11 @@ copy(
 copy_part_of_thrid_party(inference_lib_dist ${PADDLE_INFERENCE_INSTALL_DIR})
 
 set(src_dir "${PADDLE_SOURCE_DIR}/paddle/fluid")
-
+set(paddle_common_lib ${PADDLE_BINARY_DIR}/paddle/common/libcommon.*)
```
Comment (yuanlehome) [inline cmake/inference_lib.cmake]: win下叫libcommon.dll? 有lib前缀吗

Comment (yuanlehome) [reply cmake/inference_lib.cmake]: image

Comment (risemeup1) [reply cmake/inference_lib.cmake]: ok
我改一下

**Review by yuanlehome — DISMISSED** [review]
LGTM

**Review by chalsliu — DISMISSED** [review]

**Review by phlrain — APPROVED** [review]

**Pull request merged**